gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/resources/organizations/ReportersResource.java

```diff
@@ -0,0 +1,137 @@
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.am.management.handlers.management.api.resources.organizations;
+
+import io.gravitee.am.identityprovider.api.User;
+import io.gravitee.am.management.handlers.management.api.resources.AbstractResource;
+import io.gravitee.am.management.service.ReporterPluginService;
+import io.gravitee.am.management.service.ReporterServiceProxy;
+import io.gravitee.am.model.Acl;
+import io.gravitee.am.model.Reference;
+import io.gravitee.am.model.ReferenceType;
+import io.gravitee.am.model.Reporter;
+import io.gravitee.am.model.permissions.Permission;
+import io.gravitee.am.service.OrganizationService;
+import io.gravitee.am.service.exception.OrganizationNotFoundException;
+import io.gravitee.am.service.model.NewReporter;
+import io.gravitee.common.http.MediaType;
+import io.reactivex.rxjava3.core.Single;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.container.AsyncResponse;
+import jakarta.ws.rs.container.ResourceContext;
+import jakarta.ws.rs.container.Suspended;
+import jakarta.ws.rs.core.Context;
+import jakarta.ws.rs.core.Response;
+
+import java.net.URI;
+
+@Tag(name = "reporter")
+public class ReportersResource extends AbstractResource {
+    @Context
+    private ResourceContext resourceContext;
+    private final ReporterServiceProxy reporterService;
+    private final ReporterPluginService reporterPluginService;
+    private final OrganizationService organizationService;
+
+    @Inject
+    public ReportersResource(ReporterServiceProxy reporterService,
+                             ReporterPluginService reporterPluginService,
+                             OrganizationService organizationService) {
+        this.reporterService = reporterService;
+        this.reporterPluginService = reporterPluginService;
+        this.organizationService = organizationService;
+    }
+
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(summary = "List registered reporters for a security domain",
```

Comment:
TODO: can you please add an operationId for all the method to avoid collision with the reporters operation on domain level during code generation based on the OpenAPI Specification ?